src/TextUI/Command.php

```diff
@@ -1111,6 +1117,7 @@ protected function showHelp(): void
   --disallow-test-output      Be strict about output during tests
   --disallow-resource-usage   Be strict about resource usage during small tests
   --enforce-time-limit        Enforce time limit based on test size
+  --default-time-limit=<sec>  Maximum running time in seconds for unsized tests
```

Comment:
I am not perfectly happy with that help text: (a) I personally think something like "Timeout in seconds ..." is more common? (b) Is "unsized tests" also understandable for others? How short/compact must the help text be: is "... for tests without either `@small`, `@medium` or `@large`" too long?
(Sorry, if this is too picky)

Reply:
First of all: feel free to be as picky as you want, it was your idea after all!  There is no deadline pressure from management, so we can take all the time we need to get it from 'good enough' to 'exactly what I needed'. :-)

I agree with your criticism of the help text. It isn't great as I tried to get all the relevant information in less than 80 characters. Mentioning the three sizes of tests might not fit. I do agree that using _timeout_ would be nice. The config flag is already called `*-time-limit` and adding `timeout` will make it extra clear what it does. I'll commit an improved help text, let me know what you think.

Reply:
Thanks, I like this!